docs/administrator.md

```diff
@@ -649,26 +618,108 @@ A secret can be pre-provisioned in different ways:
 * Automatically provisioned via a custom K8s controller like
   [kube-aws-iam-controller](https://github.com/mikkeloscar/kube-aws-iam-controller)
 
-## Google Cloud Platform setup
+## WAL archiving and basebackups
 
-To configure the operator on GCP there are some prerequisites that are needed:
+Spilo is shipped with [WAL-E](https://github.com/wal-e/wal-e) and its successor
+[WAL-G](https://github.com/wal-g/wal-g) to perform WAL archiving. By default,
+WAL-E is used because it is more battle-tested. Additionally to the continuous
+backup stream a [basebackup](https://www.postgresql.org/docs/13/app-pgbasebackup.html)
+is initialized every night and 1am UTC.
+
+These are the pre-configured settings in the docker image:
+
+```bash
+archive_command:  `envdir "{WALE_ENV_DIR}" {WALE_BINARY} wal-push "%p"`
+restore_command:  `envdir "{{WALE_ENV_DIR}}" /scripts/restore_command.sh "%f" "%p"`
+
+AWS_ENDPOINT:         'https://s3.AWS_REGION.amazonaws.com:443'
+BACKUP_NUM_TO_RETAIN: 5
+BACKUP_SCHEDULE:      '00 01 * * *'
+USE_WALG_BACKUP:      false (not set)
+USE_WALG_RESTORE:     false (not set)
+WALE_S3_ENDPOINT:     'https+path://s3.AWS_REGION.amazonaws.com:443'
+WALE_S3_PREFIX:       's3://bucket-name/very/long/path'
```

Comment:
There is also `WALG_S3_PREFIX`, which is set to the same value.

But, depending on what cloud storage is used, it could be not WAL?_S3_PREFIX, but WAL?_GS_PREFIX, WALG_SSH_PREFIX, and so on.